javascript/javascript1/week2/other_task.js

```diff
@@ -0,0 +1,176 @@
+
+
+// Event application
+function getEventWeekday(day){
+  const weekDays=['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'];
+  const currentDate  = new Date();
+  const dayNumbers = currentDate.getDay();
+   const futureDay= (day + dayNumbers) % 7;
+   console.log(weekDays[futureDay]);
```

Comment:
Excellent use of remainder! 😄 Developers using your function might be a bit surprised that a function called `getX` does not return anything; maybe it should be up to the caller to decide whether to log it or not? 📞